ONWServices.Tests/Integration/Repositories/GameRepositoryTest.cs

```diff
@@ -76,6 +76,18 @@ public void TestDeleteByGameId()
             Assert.AreEqual(2, newResults.Count);
         }
 
+        [TestMethod]
+        public void TestGetStatusById()
+        {
+            Guid gameIdToFind = Guid.NewGuid();
+            _collection.InsertOne(new Game { GameId = Guid.NewGuid()});
+            _collection.InsertOne(new Game { GameId = gameIdToFind, Status = GameStatus.Closed });
+            _collection.InsertOne(new Game { GameId = Guid.NewGuid() });
+
+            Game result = cut.FindByGameId(gameIdToFind);            
+            Assert.AreEqual(GameStatus.Closed, result.Status);
+        }
```

Comment:
I don't think it's necessary to add a new test just to check that the status is saved properly. You can just augment the `TestFindByGameId` test above to set statuses and verify they're returned.